### ISSUE TITLE
Unable to use this plugin

### ISSUE THREAD

**Issue by chetanchitte** [Stephn-R/atom-section-comments-snippets#5]
Hello I have installed this package. I am using Macbook pro, atom I am trying to run these snippets but they aren't working.

Any idea how can I use them ?

Regards,
Chetan

**Comment by stephnr** [comment]
I will have to review. It has been some time since I have updated this plugin.

**Comment by thursdaydan** [comment]
Yeah, not sure if related, but fwiw I just started to use it in a blade file and it only allows me to use blade style comments, at the time I was trying to added header comments in javascript.
